modules/apigee/main.tf

```diff
@@ -91,7 +92,7 @@ resource "google_apigee_instance" "instances" {
   description              = each.value.description
   location                 = each.key
   org_id                   = local.org_id
-  ip_range                 = "${each.value.runtime_ip_cidr_range},${each.value.troubleshooting_ip_cidr_range}"
+  ip_range                 = var.organization.disable_vpc_peering ? null : "${each.value.runtime_ip_cidr_range},${each.value.troubleshooting_ip_cidr_range}"
```

Comment:
Same as before, I'd add a validation to variable (or maybe here you need a precondition check since it's s different variable)